template.yml

```diff
@@ -40,62 +49,65 @@ Resources:
       CodeUri: ./backend-lambda
       Environment:
         Variables:
-          DATABASE_URL: !Ref DATABASE_URL
+          DATABASE_URL: !Ref DatabaseUrl
       Events:
         Api:
           Type: Api
           Properties:
-            Path: /myendpoint
-            Method: get
+            Path: /
+            Method: ANY
 
   RestApi:
     Type: AWS::Serverless::Api
     Properties:
       StageName: prod
       Auth:
-        DefaultAuthorizer: IPAuthorizerLambdaFunction
+        DefaultAuthorizer: IpAuthorizerLambdaFunction
       DefinitionBody:
         swagger: "2.0"
         info:
           title: "RestApi"
         paths:
-          /myendpoint:
-            get:
+          /products:
+            get: &getProducts
```

Comment:
Rename to `&standardImplementation` and reuse.

Reply:
done